test/utils.js

```diff
@@ -1,26 +1,46 @@
 const expect = require('chai').expect;
+const fs = require('fs')
+const sinon = require('sinon')
 module.exports = {
-    generateTests: (parser, TestCase) => {
-        const funs = Object.keys(TestCase);
-
+    generateTests: (parser, TestCase, type) => {
+        let clock = null;
+            now = new Date();
```

Comment:
Use 2020-01-01T00:00:00Z as now so it's determinstic, it's the whole reason why we're mocking out the clock